src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsresettlementpassportapi/data/DocumentCategory.kt

```diff
@@ -1,5 +1,11 @@
 package uk.gov.justice.digital.hmpps.hmppsresettlementpassportapi.data
 
+import uk.gov.justice.digital.hmpps.hmppsresettlementpassportapi.jpa.entity.DocumentsEntity
+
 enum class DocumentCategory(val displayName: String) {
   LICENCE_CONDITIONS("Licence Conditions"),
 }
+
+data class DocumentResponse(
```

Comment:
Is this only used in tests? If so might be better to put this in the test class instead.

Reply:
Sure will consider this in my next commit!